service/src/PlatformIO.cpp

```diff
@@ -1419,4 +1424,8 @@ extern "C" {
     {
         geopm::platform_io_reset();
     }
+
+    int geopm_pio_check_valid_value(double value) {
+        return geopm::platform_io().is_valid_value(value) ? 0 : -1;
```

Comment:
Should this return a value of `enum geopm_error_e`?

Reply:
`GEOPM_ERROR_INVALID`